www/widgets/rt.js

```diff
@@ -325,6 +325,24 @@ static urllib() {
     return { parse: parse };
 }
 
+static html() {
```

Comment:
What does this new method do?

Reply:
It implements Python’s html.escape function by leveraging the browser’s html unpaused.

Reply:
Is that a well-documented technique? If so, can you add a comment to documentation for it?